libraries/base.rb

```diff
@@ -227,7 +231,11 @@ def merge_members(obj)
     end
 
     def push_object(obj, hash)
-      if hash[obj.to_s].nil?
+      return if hash.key?('null')
+      if obj.to_s == 'null'
+        hash.delete_if { |_k, _v| true }
```

Comment:
I realized that removing a link this way would probably only remove the link in one direction and not both directions. I was thinking a corresponding pop method that would keep the data integrity by removing the links from both objects would probably be better.